src/App.tsx

```diff
@@ -62,6 +63,19 @@ export interface AggData {
   all_filtered_by_year: StatisticData[];
 }
 
+function validate_treatment_units(
+  treatment_units: string[],
+  valid_treatment_units: TreatmentUnit[]
+) {
+  return (
+    (treatment_units
+      ?.filter((x) =>
+        valid_treatment_units.some((tu) => tu.hospital === x || tu.hf === x)
```

Comment:
What about rhf?

Reply:
RHFs are not selectable in the "show all screen", assumed this excludes them from query params. 

![image](https://user-images.githubusercontent.com/1658284/105717405-5edbfe00-5f20-11eb-9442-a19fa8ca502e.png)

Reply:
They are selectable from the search box:
![image](https://user-images.githubusercontent.com/136346/105718103-2c7ed080-5f21-11eb-8589-def1a2c9965e.png)

Reply:
Ah, then I assumed wrong - I'll add RHFs

Reply:
Done.